guidelines/design-guidelines.md

```diff
@@ -31,4 +44,154 @@ Example:
 var transportConfig = endpointConfig.UseTransport<MsmqTransport>();
 
 transportConfig.MsmqLabelGenerator(context => return $"{context.Headers['NServiceBus.EnclosedMessageTypes']}");
-```
\ No newline at end of file
+```
+
+### FAQ
```

Comment:
I don't really like that FAQ section. Can we either remove it or transform this into a statement to guide using AppSettings & Co. for settings which should be changeable during runtume or without recompiling?